examples/rl/racing/reward.py

```diff
@@ -1,7 +1,7 @@
 import gym
 import numpy as np
 
-from smarts.core.sensors import Observation
+from smarts.core.observations import Observation
```

Comment:
The `examples/rl/racing` setup.py uses SMARTS version 0.7.0rc0 which might need to be upgraded and the example should be re-checked with any changes to the example.